### PR TITLE
feat(related_groups): Move same title issues below similar stack traces

### PR DESCRIPTION
**Pull request by armenzg**
Similar Issues changes:
* Show "issues with similar stack trace" before "issues with same root cause"
* Rename "issues with same root cause" with "issues with similar titles"
* Added description to the section
* Removed events, users and assignee
* Reduce title font

Next steps:
* Bring further UI & style consistency between the two sections
* Refactor the code into the same directory

Before:
image
After:
image

Narrow view:
image

Wide view:
image

### CODE REVIEW

File: static/app/views/issueDetails/groupRelatedIssues/index.tsx [hunk 1]
```diff
@@ -124,28 +105,69 @@ function RelatedIssuesSection({
...
+            withColumns={[]}
           />
         </Fragment>
       ) : null}
-    </HeaderWrapper>
+    </Fragment>
   );
 }
 
+const getTraceConnectedContent = (
```
Comment (armenzg) [inline static/app/views/issueDetails/groupRelatedIssues/index.tsx:131]: The code here is the same as what I grabbed from above.

File: static/app/views/issueDetails/groupRelatedIssues/index.tsx [hunk 2]
```diff
@@ -124,28 +105,69 @@ function RelatedIssuesSection({
...
+      {t('Open in Issues')}
+    </LinkButton>
+  );
+};
+
 // Export the component without feature flag controls
 export {GroupRelatedIssues};
 
 const Title = styled('h4')`
+  font-size: ${p => p.theme.fontSizeLarge};
```
Comment (armenzg) [inline static/app/views/issueDetails/groupRelatedIssues/index.tsx:170]: Make it the same size as issue with similar stack traces header.

File: static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx [hunk 3]
```diff
@@ -193,7 +193,7 @@ function SimilarStackTrace({project}: Props) {
       {status === 'ready' && !hasSimilarItems && !hasSimilarityEmbeddingsFeature && (
         <Panel>
           <EmptyStateWarning>
-            <p>{t("There don't seem to be any similar issues.")}</p>
+            <Title>{t("There don't seem to be any similar issues.")}</Title>
```
Comment (armenzg) [inline static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx:196]: This:
image

vs this:
image

Comment (jangjodi) [reply static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx:196]: We should change the font for similarity embeddings as well [here](https://github.com/getsentry/sentry/pull/81177/files#diff-ed4ce224039f8283b042b51a4e06a71c416182eda6b4a5185e4ba3faa72ce904R205) to match

Comment (jangjodi) [reply static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx:196]: We should change the font for similarity embeddings as well [here](https://github.com/getsentry/sentry/pull/81177/files#diff-ed4ce224039f8283b042b51a4e06a71c416182eda6b4a5185e4ba3faa72ce904R205) to match

**Comment by jangjodi** [comment]
Just wondering, why are we removing events and users? I think these counts would be useful for a user to determine the the impact of a group, which may affect their decision to merge or not.

**Comment by armenzg** [comment]
> Just wondering, why are we removing events? I think these counts would be useful for a user to determine the the impact of a group, which may affect their decision to merge or not.
> 
> Also do you have a screenshot for the ui for the 'similarity-embeddings' feature flag? This ui should already not have the users columns

I removed the columns just so it would look a little less crowded but perhaps that can be handled later with some UI improvements.

I will DM me about the flag.

**Review by jangjodi — APPROVED** [review]
looks good! we chatted about keeping the events column offline

**Pull request merged**